src/site/content/en/angular/getting-started-with-server-side-rendering-in-angular/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: "Getting started with server-side rendering in Angular"
+subhead: |
+  Use SSR to get control of your SEO and social media previews and improve perceived performance in your Angular applications.
+hero: angular-ssr.jpeg
```

Comment:
This image isn't obviously tied to the content of the post, so I might omit it, but up to you.